{
  "done": [
    "torchvision",
    "norse",
    "fairscale",
    "pytorch_cluster",
    "pytorch_scatter"
  ],
  "in-pr": [
    "fcos",
    "pytorch_spline_conv",
    "torchsparse",
    "nvidia-apex"
  ],
  "awaiting-pr": [],
  "not-solvable": [],
  "awaiting-parents": [],
  "bot-error": [
    "torchani",
    "torch-scatter"
  ],
  "_feedstock_status": {
    "torchvision": {
      "num_descendants": 3,
      "immediate_children": [
        "fairscale",
        "fcos",
        "norse"
      ],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/torchvision-feedstock/pull/36",
      "pr_status": "clean"
    },
    "nvidia-apex": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/nvidia-apex-feedstock/pull/19",
      "pr_status": "unstable"
    },
    "pytorch_cluster": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pytorch_cluster-feedstock",
      "pr_status": null
    },
    "pytorch_scatter": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pytorch_scatter-feedstock/pull/41",
      "pr_status": "unstable"
    },
    "pytorch_spline_conv": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/pytorch_spline_conv-feedstock/pull/18",
      "pr_status": "unstable"
    },
    "torchani": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "bot error (<a href=\"https://github.com/regro/autotick-bot/actions/runs/1905768394\">bot CI job</a>): master: Traceback (most recent call last):\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/auto_tick.py\", line 1135, in _run_migrator\n    migrator_uid, pr_json = run(\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/auto_tick.py\", line 270, in run\n    eval_cmd(\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/utils.py\", line 276, in eval_cmd\n    c = subprocess.run(\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 507, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 1134, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 1980, in _communicate\n    self._check_timeout(endtime, orig_timeout, stdout, stderr)\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 1178, in _check_timeout\n    raise TimeoutExpired(\nsubprocess.TimeoutExpired: Command 'conda smithy rerender -c auto --no-check-uptodate' timed out after 300 seconds\n"
    },
    "torchsparse": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/torchsparse-feedstock/pull/14",
      "pr_status": "clean"
    },
    "fcos": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/fcos-feedstock/pull/4",
      "pr_status": "clean"
    },
    "torch-scatter": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "bot error (<a href=\"https://github.com/regro/autotick-bot/actions/runs/1906293139\">bot CI job</a>): master: Traceback (most recent call last):\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/auto_tick.py\", line 1135, in _run_migrator\n    migrator_uid, pr_json = run(\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/auto_tick.py\", line 270, in run\n    eval_cmd(\n  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/utils.py\", line 276, in eval_cmd\n    c = subprocess.run(\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 507, in run\n    stdout, stderr = process.communicate(input, timeout=timeout)\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 1134, in communicate\n    stdout, stderr = self._communicate(input, endtime, timeout)\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 1980, in _communicate\n    self._check_timeout(endtime, orig_timeout, stdout, stderr)\n  File \"/usr/share/miniconda3/envs/run_env/lib/python3.9/subprocess.py\", line 1178, in _check_timeout\n    raise TimeoutExpired(\nsubprocess.TimeoutExpired: Command 'conda smithy rerender -c auto --no-check-uptodate' timed out after 300 seconds\n"
    },
    "norse": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/norse-feedstock/pull/2",
      "pr_status": "unknown"
    },
    "fairscale": {
      "num_descendants": 0,
      "immediate_children": [],
      "pre_pr_migrator_status": "",
      "pr_url": "https://github.com/conda-forge/fairscale-feedstock",
      "pr_status": null
    }
  }
}